src/main/java/micdoodle8/mods/galacticraft/core/proxy/CommonProxyCore.java

```diff
@@ -14,7 +15,7 @@
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.world.World;
 
-public class CommonProxyCore {
+public class CommonProxyCore implements ISpaceProject {
```

Comment:
Why does this need to implement ISpaceProject?

Reply:
Got it confused with an artefact in GT5u. gonna remove it